bookwyrm/book_search.py

```diff
@@ -133,6 +136,32 @@ def search_title_author(query, min_confidence, *filters, return_first=False):
     return list_results
 
 
+def isbn_check(query):
+    n = query.replace('-','')
```

Comment:
Does this need a check to make sure all the characters are numeric after replacing them? Will it throw for 13-character strings that are non-numeric due to use of `int()`?

Reply:
Yes and No :-)
I'm not checking for numeric values here, but whether the 13-digit/10-digit value is an ISBN. I need int() to vivisect this (see [ISBN](https://en.wikipedia.org/wiki/International_Standard_Book_Number)). The passed value is always a string. I check for ISBN because someone should also be able to enter a title like 1950-2000.

Reply:
I just tested and indeed, if you search for `fffffffffffff` you get

```
ValueError at /search

invalid literal for int() with base 10: 'f'
```

after line 140, you should add

```
if not n.isnumeric():
    return False
```

or similar.